api/src/pcapi/routes/backoffice/offerers/offerer_blueprint.py

```diff
@@ -1027,11 +1029,87 @@ def get_managed_venues(offerer_id: int) -> utils.BackofficeResponse:
 
     return render_template(
         "offerer/get/details/managed_venues.html",
+        offerer_id=offerer_id,
         rows=rows,
         connect_as=connect_as,
     )
 
 
+def _render_get_create_venue_without_siret_form(form: pro_forms.CreateVenueWithoutSIRETForm, offerer_id: int) -> str:
+    return render_template(
+        "components/turbo/modal_form.html",
+        information="Ce formulaire permet de créer un nouveau partenaire culturel rattaché au SIRET choisi.",
+        form=form,
+        dst=url_for("backoffice_web.offerer.create_venue", offerer_id=offerer_id),
+        div_id="create-venue-modal",  # must be consistent with parameter passed to build_lazy_modal
+        title="Créer un partenaire culturel sans SIRET",
+        button_text="Créer le partenaire culturel",
+        data_turbo=True,
+    )
+
+
+@offerer_blueprint.route("/create-without-siret", methods=["GET"])
+@utils.permission_required(perm_models.Permissions.CREATE_PRO_ENTITY)
+def get_create_venue_without_siret_form(offerer_id: int) -> utils.BackofficeResponse:
+    offerer = db.session.query(offerers_models.Offerer).get_or_404(offerer_id)
```

Comment:
Comme tu boucles sur `offerer.managedVenues` dans `CreateVenueWithoutSIRETForm`, je pense qu'on peut pré-charger en une seule requête : 
```suggestion
    offerer = (
        db.session.query(offerers_models.Offerer)
        .options(sa_orm.joinedload(offerers_models.Offerer.managedVenues))
        .filter(offerers_models.Offerer.id == offerer_id)
        .one_or_none()
    )
    if not offerer:
        raise NotFound()
```

Idem (donc requête potentiellement factorisable) dans l'endpoint `create_venue` en-dessous.

Reply:
1. numéro de téléphone et site web ne sont pas utiles dans notre cas, c'est OK
2. le compte bancaire pourrait être différent dans le cas d'une collectivité, on ne souhaite pas l'ajouter (contrairement au SIRET qui lui est sélectionné dans la modale)
3. Je n'avais pas en tête la partie validation manuelle ! Bien vu, si la venue est sélectionnée, par la fraude, je pense qu'il n'y a pas besoin, de la sélectionnée ensuite, mais je pose la question ! A voir si ça met du temps à avoir la réponse côté fraude, on peut lancer et faire un ticket séparé, je vous redis

Reply:
Retour de JB : pas besoin d'avoir la venue en full validation : c'est OK !